### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by zwwhdls**
add csi upgrade doc and use doc of new version

### CODE REVIEW

**Review by yujunz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,9 +35,17 @@ storageClasses:
     name: "<name>"
     metaurl: "<redis-url>"
     storage: "<storage-type>"
-    access-key: "<access-key>"
-    secret-key: "<secret-key>"
+    accessKey: "<access-key>"
```
Comment (davies) [inline README.md:38]: Should we keep these backward compatible?

File: README.md [hunk 2]
```diff
@@ -115,12 +123,24 @@ curl -sSL https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/
 
 ## Upgrade CSI Driver
 
+### After v0.10.0
```
Comment (davies) [inline README.md]: v0.10+

File: README.md [hunk 3]
```diff
@@ -115,12 +123,24 @@ curl -sSL https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/
...
+
+Juicefs CSI Driver separated JuiceFS client from CSI Driver since v0.10.0, CSI Driver upgrade will not interrupt existing PVs. If CSI Driver version >= v0.10.0, do operations below:
+
+* If you're using `latest` tag, simple run `kubectl rollout restart -f k8s.yaml` and make sure `juicefs-csi-controller` and `juicefs-csi-node` pods are restarted.
+* If you have pinned to a specific version, modify your `k8s.yaml` to a newer version, then run `kubectl apply -f k8s.yaml`. 
+* Alternatively, if JuiceFS CSI driver is installed using Helm, you can also use Helm to upgrade it.
+
+If you want to upgrade CSI Driver from v0.9.0 to v0.10.0, follow [how to upgrade csi-driver from v0.9.0 to v0.10.0](./docs/upgrade-csi-driver.md).
+
+### Before v0.9.0
```
Comment (davies) [inline README.md]: Before v0.10

File: docs/upgrade-csi-driver.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
+# How to upgrade juicefs CSI Driver from v0.9.0 to v0.10.0
+
+Juicefs CSI Driver separated JuiceFS client from CSI Driver since v0.10.0. But the upgrade from v0.9.0 to 0.10.0 will cause all PVs become unavailable, we can upgrade one by one node to make the upgrade smooth.
```
Comment (davies) [inline docs/upgrade-csi-driver.md]: Added a section to upgrade whole cluster:

1. stop all applications
2. upgrade CSI driver
3. start all applications

File: README.md (hunk 1 above)
Comment (zwwhdls) [reply README.md:38]: The keys are `accessKey` and `secretKey` in values.yaml and I modified them wrong in https://github.com/juicedata/juicefs-csi-driver/pull/101, and fix it here 😂.

**Review by davies — APPROVED** [review]

**Pull request merged**